examples/reference-erc4626/README.md

```diff
@@ -0,0 +1,76 @@
+# Example ERC4626 token API
```

Comment:
Build is failing because our build step for the `create-ponder` templates expects a `.env.example` file in each example directory. You should add one like this (the same one as `reference-erc20`).

```
# Mainnet RPC URL used for fetching blockchain data. Alchemy is recommended.
PONDER_RPC_URL_1=https://eth-mainnet.g.alchemy.com/v2/...

# (Optional) Postgres database URL. If not provided, SQLite will be used. 
DATABASE_URL=
```